src/content/blog/2024/reflections-on-nextflow-mentorship.md

```diff
@@ -0,0 +1,78 @@
+Finally, I was honored to become a Nextflow ambassador. The program’s goal is to extend the awareness of Nextflow around the world while also building a supportive community. In particular, the South American community is underrepresented, so I serve as a point of contact for any institution or newcomer who wants to implement pipelines with Nextflow.
+As part of this program, I was invited to speak at the second Chilean Congress of Bioinformatics, where I gave a talk about how Nextflow and nf-core can support scaling bioinformatics projects in the cloud. It was incredibly rewarding to introduce Nextflow to a community for the first time and witness the genuine enthusiasm it sparks among students and attendees for the potential in their research projects.
+
+<div style="margin-top: 2rem; margin-bottom: 2rem;">
+    <img src="/img/blog-2024-04-10-img1c.png" alt="Second Chilean Congress of Bioinformatics" />
+</div>
+
+# What’s next?
+
+The comprehensive skill set acquired in my journey resonated positively and allowed me to join the ZS Discovery Team. This organization accelerates transformation in research and early development with direct contribution to impactful bioinformatics projects with a globally distributed, multidisciplinary talented team.
+
+In addition, we organized the first Nextflow Hackathon in Argentina, fostering a space to advance our skills in workflow management collectively. It was a pleasure to see how beginners got their first PRs approved and how they interacted with the nf-core community for the first time.
+
+<div style="margin-top: 2rem; margin-bottom: 2rem;">
+    <img src="/img/blog-2024-04-10-img1d.png" alt="nf-core March 2024 Hackathon site in Argentina" />
+</div>
+
+My current (and probably future!) day-to-day work involves working and developing pipelines with Nextflow, while also mentoring younger bioinformaticians into this language. The commitment to open-source projects remains a cornerstone of my journey and I am thankful that it has provided me the opportunity to collaborate with individuals from diverse backgrounds all over the world.
+
+Whether you're interested in the mentorship program, curious about the hackathon, or simply wish to connect, feel free to reach out at the nf-core Slack!
+
+<br>
+<div class="footer-wrapper">
+<img src="/img/nextflow_ambassador_logo.svg" height="70" class="pull-right" style="padding: 1rem 1rem;">
+
+> This post was contributed by a Nextflow Ambassador, passionate individuals who support the Nextflow community. Interested in becoming an ambassador? Read more about it <a href="https://www.nextflow.io/ambassadors.html">here</a>.
```

Comment:
I love this footer but the grammar is a bit iffy. Would go full stop after "This post was contributed by a Nextflow Ambassador." then folow up with the brief definition of who ambassadors are as a separate sentence.

Reply:
Updated! 😄 

What do you think about it now?